streaming/nifi/main.tf

```diff
@@ -0,0 +1,155 @@
+#---------------------------------------------------------------
+# EKS Blueprints
+#---------------------------------------------------------------
+module "eks_blueprints" {
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints?ref=v4.22.0"
```

Comment:
Going forward as per v5 refactoring in blueprints, we should use upstream EKS modules for creating EKS clusters and node groups. See examples in blueprints repo as well in upstream for doing this.